adapters/wasm-micro-runtime.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import subprocess
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--version', action='store_true')
+parser.add_argument('--test-file', action='store')
+parser.add_argument('--arg', action='append', default=[])
+parser.add_argument('--env', action='append', default=[])
+parser.add_argument('--dir', action='append', default=[])
+
+args = parser.parse_args()
+
+if args.version:
+    subprocess.run(['iwasm', '--version'])
```

Comment:
we've recently updated adapters so custom path to executable can be provided. Could we add that change here too?